core/src/main/java/hudson/model/View.java

```diff
@@ -1318,6 +1320,16 @@ public void updateByXml(Source source) throws IOException {
     public static final Permission CONFIGURE = new Permission(PERMISSIONS,"Configure", Messages._View_ConfigurePermission_Description(), Permission.CONFIGURE, PermissionScope.ITEM_GROUP);
     public static final Permission READ = new Permission(PERMISSIONS,"Read", Messages._View_ReadPermission_Description(), Permission.READ, PermissionScope.ITEM_GROUP);
 
+    @Initializer(before = InitMilestone.SYSTEM_CONFIG_LOADED)
+    @Restricted(DoNotUse.class)
```

Comment:
That'll teach me to accept "suggestions" on GitHub.

Reply:
caveat acceptor

Reply:
I checked when I suggested and both classes are imported?

Reply:
You're not missing anything. No imports are needed, hence my joke about accepting suggestions on GitHub (the point of the joke being that even when such suggestions don't lead to compilation errors, they do often lead to unnecessary review feedback :smile:).

Reply:
Import ordering lol, they're in the exact same package yet miles away in the imports.

I just saw `NoExternalUse` and `DoNotUse` wasn't on my screen.

Reply:
> Import ordering lol, they're in the exact same package yet miles away in the imports.

I've been debating whether or not to sort all the imports in core. It would take me five minutes of course, but not sure how receptive others would be. :smile:

Reply:
me too, was just thinking of doing it via spotless

Reply:
=> #5732